src/api.ts

```diff
@@ -28,6 +28,15 @@ export interface Config {
      * - `"string"`: returns SQLite integers as strings.
      */
     intMode?: IntMode;
+
+    /** Custom `fetch` function to use for the HTTP client.
+     *
+     * By default, the HTTP client uses `fetch` from the `@libsql/isomorphic-fetch` package, but you can pass
+     * your own function here. The argument to this function will be `Request` from
+     * `@libsql/isomorphic-fetch`, and it must return a promise that resolves to an object that is compatible
+     * with the Web `Response`.
+     */
+    fetch?: unknown;
```

Comment:
Is there a way to type this more cleanly? TypeScript shows the builtin fetch function as `fetch(input: RequestInfo | URL, init?: RequestInit): Promise<Response>`.

Reply:
There is, but the `@libsql/client` library and the user would have to depend on the `@libsql/isomorphic-fetch` package to get the types, and there might be type mismatches if the user's `fetch` function came from somewhere else than the `@libsql/isomorphic-fetch` package. It seems to me that this is a case where the hassle of typing outweights the benefits.